onnxruntime/core/mlas/lib/sgemm.cpp

```diff
@@ -227,6 +227,17 @@ Return Value:
     }
 }
 
+#if defined(MLAS_TARGET_WASM) and !defined(MLAS_TARGET_WASMSIMD)
```

Comment:
as discussed in teams, i don't like how this change is structured: i'd rather see two versions of the funciton and not try to #ifdef the existing code.

Reply:
changed.